source/main.cpp

```diff
@@ -568,6 +580,9 @@ int main(int argc, char *argv[])
                 break;
             case SIGHUP:
                 resourceManager->dumpMemTrace();
+                FeatureRegistry reg;
```

Comment:
This is weird. You have to call stop on the registry with features in it, not this temporary.

Reply:
That was left overs from testing, I forgot to delete in the push.